src/api/APIFactory.js

```diff
@@ -252,6 +258,20 @@ class APIFactory {
         this.versionsAPI = new VersionsAPI(this.options);
         return this.versionsAPI;
     }
+
+    /**
+     * API for fille access stats
+     *
+     * @param {boolean} shouldDestroy - true if the factory should destroy before returning the call
+     * @return {FileAccessStatsAPI} FileAccessStatsAPI instance
+     */
+    getFileAccessStatsAPI(shouldDestroy: boolean): FileAccessStatsAPI {
```

Comment:
should shouldDestroy be defaulted to true?

Reply:
Hmm I am not sure about that. Destroying the factory destroys the cache. A call to get access stats shouldnt destroy the cash

Reply:
Factory isn't destroyed, I think he meant that the individual API is itself destroyed by default.